### ISSUE TITLE
Error in macro F1

### ISSUE THREAD

**Issue by djstrong** [zalandoresearch/flair#486]
Probably there is error in calculation of macro F1. Flair is taking averages of class precisions and recalls and then calculates F1. As far as I know, it should be F1 calculated over classes and then averaged - it would be consistent with `sklearn.metrics.f1_score(labels, predicitons, average='macro')`.

**Comment by alanakbik** [comment]
fixed in #521
